src/pytorch_lightning/strategies/fully_sharded_native.py

```diff
@@ -115,7 +111,7 @@ def __init__(
         checkpoint_io: Optional[CheckpointIO] = None,
         precision_plugin: Optional[PrecisionPlugin] = None,
         process_group_backend: Optional[str] = None,
-        cpu_offload: Optional[CPUOffload] = None,
+        cpu_offload: Optional[Union[bool, "CPUOffload"]] = None,
```

Comment:
```suggestion
        cpu_offload: Union[bool, "CPUOffload", None] = None,
```
less brackets are easier to read... :rabbit: